src/devices/Common/Iot/Device/Common/WeatherHelper.cs

```diff
@@ -138,11 +138,36 @@ public static Temperature CalculateDewPoint(Temperature airTemperature, Ratio re
         /// <remarks>
         /// Source https://de.wikipedia.org/wiki/Luftfeuchtigkeit#Absolute_Luftfeuchtigkeit
         /// </remarks>
-        public static double CalculateAbsoluteHumidity(Temperature airTemperature, Ratio relativeHumidity)
+        public static Density CalculateAbsoluteHumidity(Temperature airTemperature, Ratio relativeHumidity)
```

Comment:
I think there was an open issue to fix this to return units right?, if this was the only one missing lets make sure to close that issue with this PR

Reply:
#1181 is actually about a different method. I think I shouldn't touch that, since somebody else is fixing that.

Reply:
if there is no open PRs there then you're good, make sure to update the issue 😄